src/Common/test-runtime/LatestDependencies/project.json

```diff
@@ -0,0 +1,75 @@
+{
```

Comment:
Is something like "project.json.floating", which gets renamed when we need it possible? 

In my mind, it's just crappy to see another project.json file being maintained for no great reason, but I gues we just find+replace across all files anyway, and it's not too bad. 

I'm just being a bit of a devil's advocate here to see if there's something cleaner 
Your call on what you like best - I'm done being pedantic and will not pursue this any further :smile: